src/data/fin.lean

```diff
@@ -40,6 +42,12 @@ instance fin_to_nat (n : ℕ) : has_coe (fin n) nat := ⟨fin.val⟩
 
 lemma coe_eq_val (a : fin n) : (a : ℕ) = a.val := rfl
 
+@[simp] lemma val_one  {n : ℕ} : (1 : fin (n+2)).val = 1 := rfl
```

Comment:
Presumably `val_zero` is somewhere else?

Reply:
yes, in core.